lib/repl_type_completor/types.rb

```diff
@@ -26,7 +26,15 @@ def self.load_rbs_builder
       @load_started = true
       loader = RBS::CLI::LibraryOptions.new.loader
       loader.add path: Pathname('sig')
-      @rbs_builder = RBS::DefinitionBuilder.new env: RBS::Environment.from_loader(loader).resolve_type_names
+      env = RBS::Environment.from_loader(loader)
+      # Improve development experience
+      env.extend(Module.new {
+        def resolve_declaration(*)
```

Comment:
I think `(*, **)` is more safe, considering new keyword parameter addition in new version of RBS.

Using singleton method like this is more simple, though it is not identical to `env.extend(Module.new{})`
```ruby
def env.resolve_declaration(*, **)
end
```

Thread.pass approach looks good :+1:

Reply:
Fixed!